testing/screenshots/index.js

```diff
@@ -5,43 +5,49 @@ const puppeteer = require("puppeteer");
+                }
+                let pullRequest;
+                if (context.payload.inputs.pr) {
+                    pullRequest = parseInt(context.payload.inputs.pr);
+                }
+                return {
+                    files,
+                    pullRequest
+                };
             } else {
-                files = context.payload.inputs.files.split(/,\s?/).map(x => path.resolve(dashboardsBase, x + ".json"));
-            }
-            let pullRequest;
-            if (context.payload.inputs.pr) {
-                pullRequest = parseInt(context.payload.inputs.pr);
+                const base = context.payload.pull_request.base.sha;
+                const head = context.payload.pull_request.head.sha;
+                // Ignore deleted files
+                const changedFiles = child_process.execSync(`git diff --diff-filter=d --name-only ${base} ${head} `, { encoding: "ascii" }).trim().split("\n");
+                return {
+                    files: changedFiles,
+                    pullRequest: context.payload.pull_request.number
+                };
             }
-            return {
-                files,
-                pullRequest
-            };
-        } else {
-            const base = context.payload.pull_request.base.sha;
-            const head = context.payload.pull_request.head.sha;
-            // Ignore deleted files
-            const changedFiles = child_process.execSync(`git diff --diff-filter=d --name-only ${base} ${head}`, { encoding: "ascii" }).trim().split("\n");
-            return {
-                files: changedFiles,
-                pullRequest: context.payload.pull_request.number
-            };
-        }
-    } else {
-        if (process.argv.length < 3) {
-            throw new Error("Insufficient arguments. Usage: node index.js [comma-separated-files | 'all'");
-        }
-        const filesInput = process.argv[2];
-        if (filesInput.trim() === "all") {
-            files = fs.readdirSync(dashboardsBase).map(x => path.join(dashboardsBase, x)).filter(x => /\.json$/.test(x));
         } else {
-            files = filesInput.split(/,\s?/).map(x => path.resolve(dashboardsBase, x + ".json"));
+            if (process.argv.length < 3) {
+                throw new Error("Insufficient arguments. Usage: node index.js [comma-separated-files | 'all']");
+            }
+            const filesInput = process.argv[2];
+            if (filesInput.trim() === "all") {
+                files = fs.readdirSync(dashboardsBase).map(x => path.join(dashboardsBase, x)).filter(x => /\.json$/.test(x));
+            } else {
+                files = filesInput.split(/,\s?/).map(x => path.resolve(dashboardsBase, x + ".json"));
+            }
+            return { files, pullRequest: undefined };
         }
+    } catch(e) {
+        console.log("Error detected so defaulting to all dashboards", e);
```

Comment:
NITPICK: this should probably be `console.warn` or `console.error` so it goes to stderr instead of stdout